Source/WebKit/WebProcess/cocoa/UserMediaCaptureManager.h

```diff
@@ -78,6 +78,7 @@ class UserMediaCaptureManager final : public WebProcessSupplement, public IPC::M
     void addSource(Ref<RemoteRealtimeVideoSource>&&);
     void removeSource(WebCore::RealtimeMediaSourceIdentifier);
 
+    Ref<RemoteCaptureSampleManager> protectedRemoteCaptureSampleManager() { return remoteCaptureSampleManager(); }
```

Comment:
Doesn't this work? That's what we'd normally do I think.
```suggestion
    Ref<RemoteCaptureSampleManager> protectedRemoteCaptureSampleManager() { return m_remoteCaptureSampleManager; }
```

Reply:
Yup that works too! I was originally torn between which version I should use haha.

Fixed.